dev/TeachingTip/TeachingTip.cpp

```diff
@@ -199,9 +199,27 @@ void TeachingTip::OnPropertyChanged(const winrt::DependencyPropertyChangedEventA
             TeachingTipTestHooks::NotifySubtitleVisibilityChanged(*this);
         }
     }
+    else if (property == s_ActionButtonContentProperty)
+    {
+        UpdateButtonsState();
+        winrt::IInspectable value = unbox_value<winrt::IInspectable>(args.NewValue());
+        UpdateButtonAutomationProperties(m_actionButton.get(), value);
+    }
+    else if (property == s_CloseButtonContentProperty)
+    {
+        UpdateButtonsState();
+        winrt::IInspectable value = unbox_value<winrt::IInspectable>(args.NewValue());
+        UpdateButtonAutomationProperties(m_closeButton.get(), value);
+    }
 
 }
 
+void TeachingTip::UpdateButtonAutomationProperties(const winrt::Button button, const winrt::IInspectable content)
+{
+    winrt::hstring nameHString = SharedHelpers::TryGetStringRepresentationFromObject(content);
```

Comment:
Add a null check for 'button' here, since someone could have retemplated and not have the button in their template.